opm/parser/eclipse/EclipseState/Schedule/Well.hpp

```diff
@@ -148,7 +150,7 @@ namespace Opm {
 
         WellCompletion::CompletionOrderEnum m_comporder;
         bool m_allowCrossFlow;
-
+        MessageContainer m_messages;
```

Comment:
My oh my - that `MessageContainer` is spreading *everywhere*, but that is not for this PR to consider.